### DISCUSSION TITLE
Change default kustomize version

### DISCUSSION THREAD

**Discussion by driv** [argoproj/argo-cd#14270] [Q&A]
I'm migrating from argocd 2.6.0 to 2.7.6.

I'm having a sync failure on most applications because they are not compatible with kustomize 5.

I'm aware that I can configure a different version of kustomize and set it per application, but I would like to avoid that and just have argocd use 4.2.0 by default.

Is there any way to do this through configuration? I could not find in the documentation.

**Comment by pokgak** [comment]
for anyone stumbling into this: see https://argo-cd.readthedocs.io/en/stable/operator-manual/custom_tools/
